### ISSUE TITLE
Place all info/success messages in flash

### ISSUE THREAD

**Issue by victorbrender** [Learn-by-doing/bitstarter#82]
~~Use https://www.npmjs.com/package/express-flash~~
@chill117  Can you confirm that this package should be used to implement flash in the application? Thank you!

**Comment by chill117** [comment]
@victorbrender No, I wouldn't use that module because it hasn't been maintained for 3 years. Flash messages are actually pretty simple. I don't think it would be too difficult to implement this without a module.

**Comment by bokova** [comment]
@chill117  "Place *all* info..." - would help if the task was more specific as to what exactly should be in a flash message

**Comment by victorbrender** [comment]
Flash messages proposal:

- [ ] After login -> You have successfully logged in as 'Stan Marsh'
- [ ] After logout -> You have successfully logged out
- [ ] After successfully creating a project -> You have successfully created project 'Acme'
- [ ] After successfully editing a project -> Project 'Acme' was successfully updated
- [ ] After an error while editing a project -> There was an error while updating project 'Acme'
- [ ] After deleting a project -> Project 'Acme' was deleted
- [ ] After successfully editing your profile -> Your profile was successfully updated
- [ ] After an error while editing your profile -> There was an error while updating your profile
